scripts/lib/workspace_command.py

```
@@ -0,0 +1,778 @@
+            if not path.is_dir():
+                msg = f"{path}: entrada invalida em {scripts_root}"
+                raise RegistryError(msg)
+            if path.name in IGNORED_DIRS:
+                continue
+            _discover_verb_dir(registry, path)
+    registry.validate()
+    return registry
+
+
+def _script_roots() -> list[Path]:
+    return [SCRIPTS, *sorted(SUBMODULE_SCRIPT_ROOTS, key=lambda p: str(p))]
```

Ruff (0.16.1)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)